packages/fxa-shared/payments/stripe.ts

```diff
@@ -263,6 +268,16 @@ export abstract class StripeHelper {
           PAYMENT_METHOD_RESOURCE
         );
     }
+    if (expand?.includes('tax')) {
+      const customerWithTax = await this.stripe.customers.retrieve(
+        customer.id,
+        { expand: ['tax'] }
+      );
+      if (customerWithTax.deleted) {
+        return;
+      }
+      customer.tax = customerWithTax.tax;
```

Comment:
This seems suboptimal -- I guess there's no `tax` resource we can expand separately, and we don't store the Stripe customer in Firestore with the expanded tax property. Should we?

We're now making an extra Stripe API call every time we create a new Stripe subscription (when automatic tax is enabled).

Reply:
If we were to expand it in Firestore, we'd be creating a migration situation where we'd either need to go through every customer record in Stripe to update it to include tax, or update loading code to always check that, and re-load and refresh Firestore in that case.

Not only is our current checkout page sub-optimal, its buggy with state on both sides. When we refactor to do a single checkout call, the create customer will happen along with the subscription creation, which will allow it to get back the tax expanded. In the mean-time, it gets worse. :(